packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -649,14 +644,20 @@ export function getOutputLabel(outputs: WorkflowOutput<any>[], id: string) {
 	const selectedOutput = outputs.find((output) => output.id === id);
 	if (!selectedOutput) return '';
 
+	let assetId: ProjectAsset['assetId'];
+
 	// multiple output types, choose first name to use as label arbitrarily
 	if (selectedOutput.type.includes('|')) {
 		const outputType = selectedOutput.type.split('|');
-		return useProjects().getAssetName(selectedOutput.value?.[0]?.[outputType?.[0]]) || selectedOutput.label;
+		assetId = selectedOutput.value?.[0]?.[outputType?.[0]];
+
+		// default use single output type
+	} else {
+		assetId = selectedOutput.value?.[0];
 	}
 
-	// default use single output type
-	return useProjects().getAssetName(selectedOutput.value?.[0]) || selectedOutput.label;
+	const label = useProjects().getAssetName(assetId).value;
+	return computed(() => (!isEmpty(label) ? label : (selectedOutput.label ?? '')));
```

Comment:
There seems to be a couple of extra layers of unnecessary computed going. on here, what is the problem here? The PR description is blank so I am not even sure what to test for ??